### PR TITLE
Added division functionality

### PR DESCRIPTION
**Pull request by docjon09**

### CODE REVIEW

**Comment by docjon09** [comment]
Hi @GOPAL-YADAV-D, can you review this PR?

**Comment by docjon09** [comment]
Hi @DNA5769, please check this PR.

**Comment by GOPAL-YADAV-D** [comment]
> Hi @GOPAL-YADAV-D, can you review this PR?

It is under review,

File: routes/routes.py [hunk 1]
```diff
@@ -19,8 +19,54 @@ def multiply():
...
+            }), HTTPStatus.BAD_REQUEST
+    
+    # Check if the divisor is zero
+    if data['param2'] == 0:
+        return jsonify({
+            "result": None,
+            "meta": { "error": "Division by zero is not allowed" }
+        }), HTTPStatus.BAD_REQUEST
+    
+    div_res = data["param2"] / data["param2"]
```
Comment (GOPAL-YADAV-D) [inline routes/routes.py]:
```suggestion
    div_res = data["param1"] / data["param2"]
```

Comment (GOPAL-YADAV-D) [reply routes/routes.py]: And Please also resolve the merge conflicts.

**Comment by GOPAL-YADAV-D** [comment]
@docjon09 ,Please resolve the conflicts too

**Comment by docjon09** [comment]
Hi @GOPAL-YADAV-D , I have done the required corrections. Please review them.

**Comment by GOPAL-YADAV-D** [comment]
@docjon09 ,Thank you for your valuable contribution!! . Do check out other repositories from IEEE-VIT.

**Pull request merged**